### ISSUE TITLE
The code blocks for different languages need to be changed synchronously

### ISSUE THREAD

**Issue by dongxuwang** [scala/docs.scala-lang#1339]
The non English version would not update whenever someone updates the code in some code block for the English version. For instance,
[This commit](https://github.com/scala/docs.scala-lang/commit/86b8f66fe0e78fae166b4fd39900a546df041e59) updated some lines in the code block:
https://docs.scala-lang.org/tour/pattern-matching.html
However, others did not:
https://docs.scala-lang.org/zh-cn/tour/pattern-matching.html

**Comment by SethTisue** [comment]
yes, volunteers are needed to keep the translations up to date with the English originals.

**Comment by jxnu-liguobin** [comment]
how can join? i am a  chinese scala engineer

**Comment by SethTisue** [comment]
@jxnu-liguobin you can just start sending pull requests! they don't need to be perfect or complete — any kind or amount of improvement is welcome. the repo README has info on the technical aspects of contributing. English-language help is available on https://gitter.im/scala/contributors

**Comment by martijnhoekstra** [comment]
Do we maybe need a meta issue of how to keep track of what updates have and haven't been translated to what languages, or is that overdoing it?

**Comment by SethTisue** [comment]
@martijnhoekstra My guess is that the volunteer effort to keep that updated across all languages probably doesn't exist, but volunteers might exist for doing that for particular languages.

Anyone interested in coordinating an effort around a particular language is welcome to open a language-specific issue in this repo. (I suggest one issue per language, multiples would be too noisy for repo watchers.)
